src/components/NavigationSection.svelte

```diff
@@ -74,26 +73,26 @@
       return {
         busy: false,
         closed: true,
-        valve: false
+        isFetching: true
```

Comment:
Does passing it to `true` by default not blocking any opening? (but I think we now haven't any other pane that doesn't need to be delayed)